next.config.js

```diff
@@ -1,12 +1,13 @@
-const withHashicorp = require('@hashicorp/nextjs-scripts')
+const withHashicorp = require('@hashicorp/platform-nextjs-plugin')
 const withSwingset = require('swingset')
 
 module.exports = withHashicorp({
   transpileModules: [
     'swingset',
     '@hashicorp/react-.*',
-    '@hashicorp/nextjs-scripts',
+    '@hashicorp/platform-*',
     '@hashicorp/sentinel-embedded',
+    '@hashicorp/platform-product-meta',
```

Comment:
Shouldnt the glob cover this one?

Reply:
Ah yeah, good catch.

Reply:
Ended up needing to adjust this slightly so we can handle deps in workspace packages. All seems good now!